bin/improver-wind-downscaling

```diff
@@ -113,6 +127,20 @@ def main():
     non_dim_coords = [x.name() for x in wind_speed.coords(dim_coords=False)]
     if 'realization' in non_dim_coords:
         wind_speed = iris.util.new_axis(wind_speed, 'realization')
+
+    if args.output_height_level:
```

Comment:
I've had a look at the code and it all looks good apart from one minor point.

I can see that someone could accidentally specify "output_height_level_units" and forget to specify "output_height_level". While this wouldn't prevent the code working, it might be worth adding a line so the code raises a non-fatal error. Ensuring that the user knows that they've made a mistake and can review their commands.

Reply:
Added.